lib/api/statistics.js

```diff
@@ -3,6 +3,10 @@
  * @fileOverview Module for interacting with the statistcs.json endpoint.
  *
  */
+
+// Local
+const Promise = require('../polyfills/promise');
```

Comment:
If you are doing this can you modify the readme for this module to explain that `Promise` is polyfilled but `fetch` is not?